### PR TITLE
Hello World | Category Screen

### PR DESCRIPTION
**Pull request by vaibhavnagda**
Trying to understand react - hooks and javascript 😅 

I am not committed enough. I have referred to Swapnil's POC branch, understood and took out bits relevant to set up Category Screen.

### CODE REVIEW

File: app/views/screens/CategoryListScreen.js [hunk 1]
```diff
@@ -2,28 +2,27 @@ import React from "react";
...
-import { defaultStyles, colors } from "../styles";
+import CategoryList from "../components/Category/CategoryList";
 
-function CategoryListScreen(props) {
+
+function CategoryListScreen({ navigation }) {
   return (
     <Screen style={styles.container}>
-      <Text style={styles.text}>Category list screen coming soon...</Text>
+      <Text style={defaultStyle.headerText}>Categories</Text>
```
Comment (swapnilkamat23) [inline app/views/screens/CategoryListScreen.js:11]:
```suggestion
      <Text style={styles.headerText}>Categories</Text>
```

File: app/views/components/Category/CategoryItem.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+}
+const styles = StyleSheet.create({
+    item: {
+      flex: 1,
+      margin: 3,
+      height: 130,
+      justifyContent: "center",
+      alignItems: "center",
+      borderRadius: 4,
+      backgroundColor: colors.sb_orange_200,
```
Comment (swapnilkamat23) [inline app/views/components/Category/CategoryItem.js:25]: missing sb_orange_200 in colors.ts

File: package.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@expo/webpack-config": "^18.0.1",
-    "@react-native-async-storage/async-storage": "^1.18.1",
+    "@react-native-async-storage/async-storage": "1.17.11",
```
Comment (swapnilkamat23) [inline package.json:13]: Revert

**Comment by vaibhavnagda** [comment]
Addressed in #10

**Pull request closed without merging**